qoqo_qir/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "qoqo_qir"
-version = "0.1.0"
+version = "0.1.2"
 dependencies = ['qoqo>=1.15', 'qoqo_calculator_pyo3>=1.2']
```

Comment:
Should this be updated too?

Reply:
Yes, and should be 1.0.1 already but I guess it was forgotten